modules/installing-microshift-rpm.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc 
+
+:_content-type: PROCEDURE
+[id="installing-microshift-from-rpm-package_{context}"]
+= Installing MicroShift from an RPM package 
+
+Use the following procedure to install MicroShift from an RPM package. 
+
+.Prerequisites 
+
+* The xref:../microshift_install/microshift-install-rpm.adoc#system-requirements-installing-microshift[system requirements for installing MicroShift] have been met. 
+* You have completed the steps at xref:../microshift_install/microshift-install-rpm.adoc#preparing-install-microshift-from-rpm-package_microshift-install-rpm[Preparing your system for installing MicroShift from an RPM package].
+
+.Procedure
+
+. Enable the MicroShift repositories by entering the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos \
+    --enable rhocp-4.12-for-rhel-8-$(uname -i)-rpms \
+    --enable fast-datapath-for-rhel-8-$(uname -i)-rpms
+----
+
+. Install MicroShift by entering the following command:
++
+[source,terminal]
+----
+$ sudo dnf install -y microshift
+----
+
+. Download your installation link:https://console.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat Hybrid Cloud Console]. This pull secret allows you to authenticate with the container registries that serve the container images used by MicroShift.
+
+. Copy the pull secret to your {op-system} machine the `/etc/cio/openshift-pull-secret` folder of your {op-system} machine. 
```

Comment:
I see the next steps change the owner of the file, but will a user be able to write to that directory at all if they aren't root?

Reply:
How about having them download the credentials to `$HOME/openshift-pull-secret` here (instead of trying to save it directly to the `/etc` file, then adding a step to move it into place with a command like `sudo cp $HOME/openshift-pull-secret /etc/crio/openshift-pull-secret`? Then the rest of the steps would be correct as they're written.

Reply:
There are so many ways to download and get it